test/e2e/scenarios/Snapshot/snapshot_create_terminal_condition.yaml

```diff
@@ -13,11 +13,12 @@ steps:
         name: snapshot$RANDOM_SUFFIX
       spec:
         name: snapshot$RANDOM_SUFFIX
-    wait: 600
-    expect:
+    wait:
       status:
         conditions:
-          ACK.Terminal: "True"
+          ACK.Terminal:
+            status: "True"
+            timeout: 600
```

Comment:
It takes like 2 seconds to call `CreateSnapshot` and the response is practically immediate. It shouldn't take 10 minutes to reach the condition, more like 5 seconds.

Reply:
Will create a new PR to fix all timeout.